{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -31,11 +31,11 @@ name = "{{cookiecutter.license}}"
 [project.optional-dependencies]
 dev = [
   "cruft",
-  "mypy",
+  "mypy",  
+  "pandas-stubs", # mypy stubs for pandas
   "pre-commit==2.20.0,<2.21.0",
   "ruff==0.0.254", # important that these match the pre-commit hooks
   "black[jupyter]==22.8.0", # important that these match the pre-commit hooks
```

Comment:
This re-ordering will cause merge conflicts for all downstream projects, so would prefer to omit it :-)